package.json

```diff
@@ -63,7 +65,9 @@
     "debug/package.json",
     "hooks/dist",
     "hooks/src",
-    "hooks/package.json"
+    "hooks/package.json",
+    "test-utils/src",
+    "test-utils/package.json"
```

Comment:
`test-utils/dist` should also be added to the files to include in pur `npm` package :+1: